examples/twr_aloha/README.md

```diff
@@ -2,67 +2,158 @@
+
+3. Get the experiment nodes list:
+
+``` bash
+$ iotlab-experiment get --nodes
+{
+    "items": [
+        ...
+        {
+            "archi": "dwm1001:dw1000",
+            "camera": "0",
+            "mobile": "0",
+            "mobility_type": " ",
+            "network_address": "dwm1001-2.saclay.iot-lab.info",
+            "production": "YES",
+            "site": "saclay",
+            "state": "Alive",
+            "uid": " ",
+            "x": "4.42",
+            "y": "68.3",
+            "z": "6"
+        },
+        ...
+    ]
+}
+```
 
-The different algorithm options are:
+Take note of each node's `"network_address"`, when flashing the devices set
+`IOTLAB_NODE` to the value of `"network_address"`, so in this case:
+`IOTLAB_NODE=dwm1001-2.saclay.iot-lab.info`.
 
-- UWB_DATA_CODE_SS_TWR
-- UWB_DATA_CODE_SS_TWR_EXT
-- UWB_DATA_CODE_SS_TWR_ACK
-- UWB_DATA_CODE_DS_TWR
-- UWB_DATA_CODE_DS_TWR_EXT
+##### Free up the resources
 
-### Notes
+You are done with the experiment, so stop your experiment to free up the
+devices:
 
-The application example fixes the `short_address` of the anchor to
-`ANCHOR_ADDRESS`, by default `0x1234`. All tags will send ranging requests
-to that address.
+``` bash
+$ iotlab-experiment stop
+```
 
-This value can be overridden with a `CFLAG += -DANCHOR_ADDRESS=$(VALUE)`.
+### Setup
 
-Ranging request are sent every 40ms, this value can also be overridden
-by changing the default value of `RANGE_REQUEST_T_US`, e.g:
-`CFLAG += -DRANGE_REQUEST_T_US=10000` to set it at 10ms.
+1. In two separate terminal flash two nodes and open a serial connection to the
+board, prefix with IOTLAB_NODE if using IoT-LAB
+
+    $ make -C examples/twr_aloha/ flash term -j
+    $ make -C examples/twr_aloha/ flash term -j
+
+1. On at least one node enable listening and get its short address
+
+    ```shell
+    > twr lst on
+    [twr]: start listening
+    > ifconfig
+    Iface  3        HWaddr: C8:0C  NID: DE:CA
+
+                    Long HWaddr: 08:2B:31:07:CC:52:C8:0C
+    ```
+
+1. On the second node send a request to neighbors short address:
+
+    ```shell
+    >twr req C8:0C
+    [twr]: start ranging
+    ```
+
+Both nodes should show the ranging result:
+
+    ```shell
+    > {"t": 55158, "src": 49961, "dst": 51212, "d_cm": 39}
+    ```
+
+By default `twr-ss` (Single-Sided Two-Way-Ranging) is used, try using different
+protocols, check the ones available with `help` command:
+
+    ```shell
+    > twr req --help
+    Usage:
+            twr req <short_addr> [-p <proto>] [-c <count>] [-h][-i <ms interval>]
+            - short_addr: short address of request destination
+            - count: number of requests (default: 1)
+            - ms interval: wait interval milliseconds between requests(default: 1000)
+            - proto: twr protocol (ss|ss-ext|ss-ack|ds|ds-ext, default: ss)
+            twr lst on: start listening for ranging requests
+            twr lst off: stop listening for ranging requests
+    ```
```

Comment:
```suggestion
```shell
> twr req --help
Usage:
        twr req <short_addr> [-p <proto>] [-c <count>] [-h][-i <ms interval>]
        - short_addr: short address of request destination
        - count: number of requests (default: 1)
        - ms interval: wait interval milliseconds between requests(default: 1000)
        - proto: twr protocol (ss|ss-ext|ss-ack|ds|ds-ext, default: ss)
        twr lst on: start listening for ranging requests
        twr lst off: stop listening for ranging requests
```
```